wordcloud/wordcloud.py

```diff
@@ -503,16 +508,17 @@ def process_text(self, text):
 
         flags = (re.UNICODE if sys.version < '3' and type(text) is unicode
                  else 0)
-        regexp = self.regexp if self.regexp is not None else r"\w[\w']+"
+        regexp = self.regexp if self.regexp is not None else r"(\w[\w']+|\w+)"
```

Comment:
The idea was not to add a ``render_numbers`` parameter but include numbers directly via the regexp, like this: http://stackoverflow.com/questions/3831051/regex-how-to-match-a-string-that-is-not-only-numbers